examples/xgboost/README.md

```diff
@@ -1,137 +1,22 @@
-# Tree-based Federated Learning for XGBoost 
+# Federated Learning for XGBoost 
 
 ## Introduction to XGBoost and HIGGS Data
 
 ### XGBoost
-This example shows how to use [NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html) on tabular data applications.
-It uses [XGBoost](https://github.com/dmlc/xgboost),
+These examples show how to use [NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html) on tabular data applications.
+They use [XGBoost](https://github.com/dmlc/xgboost),
 which is an optimized distributed gradient boosting library.
 
 ### HIGGS
-This example illustrates a binary classification task based on [HIGGS dataset](https://archive.ics.uci.edu/ml/datasets/HIGGS). This dataset contains 11 million instances, each with 28 attributes.
+The examples illustrate a binary classification task based on [HIGGS dataset](https://archive.ics.uci.edu/ml/datasets/HIGGS). This dataset contains 11 million instances, each with 28 attributes.
 
-## Federated Training of XGBoost under Tree-based Schemes
-Several mechanisms have been proposed for training an XGBoost model under federated learning setting. In this example, we illustrate the use of NVFlare under horizontal federated learning, and tree-based collaboration.
+## Federated Training of XGBoost
+Several mechanisms have been proposed for training an XGBoost model in a federated learning setting. In these examples, we illustrate the use of NVFlare to carry out  *horizontal* federated learning using two approaches: histogram-based collaboration and tree-based collaboration.
 ### Horizontal Federated Learning
-Under horizontal setting, each participant / client joining the federated learning will have part of the whole data / instances, while each instance has full features. This is in contrast to vertical federated learning, where each client has part of the feature values.
-### Tree-based Collaboration
-There are multiple ways to perform federated collaboration for XGBoost training, this example shows tree-based methods: the trees will be independently trained on each client's local data without collecting the global knowledge. Trained trees will be collected and passed to server / other clients for aggregation and further training.  
+Under horizontal setting, each participant / client joining the federated learning will have part of the whole data / instances / examples/ records, while each instance has all the features. This is in contrast to vertical federated learning, where each client has part of the feature values for each instance.
+#### Histogram-based Collaboration
```

Comment:
content LGTM, just confirm that this name is our conclusion?

Reply:
makes sense to me